### ISSUE TITLE
get/search entity failed using http-api

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#3929]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
[2020-10-09 14:37:21-ERROR-root]: http://192.168.1.238:19121/collections/test_f218Kfjk/entities?ids=0,1 (client.py:14)
[2020-10-09 14:37:21-INFO-root]: server=oatpp/1.1.0
code=500
description=Internal Server Error
message=[oatpp::web::protocol::http::incoming::SimpleBodyDecoder::decode()]: Error. Invalid Request.
 (client.py:18)

```
**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**

**Method of installation**
- [x] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by del-zhenwu** [comment]
search got the same error msg

**Comment by fishpenguin** [comment]
Fixed in pr#3926
